jupyter/minimal/ubi8-python-3.8/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+LABEL name="odh-notebook-jupyter-minimal-ubi8-python-3.8" \
+    summary="Minimal jupyter notebook image for ODH notebooks" \
+    description="Minimal jupyter notebook image with base Python 3.8 builder image based on UBI8 for ODH notebooks" \
+    io.k8s.display-name="Minimal jupyter notebook image for ODH notebooks" \
+    io.k8s.description="Minimal jupyter notebook image with base Python 3.8 builder image based on UBI8 for ODH notebooks" \
+    authoritative-source-url="https://github.com/opendatahub-io/notebooks" \
+    io.openshift.s2i.build.commit.ref="main" \
+    io.openshift.s2i.build.source-location="https://github.com/opendatahub-io/notebooks/tree/main/base/jupyter/minimal/ubi8-python-3.8" \
```

Comment:
Link should be `https://github.com/opendatahub-io/notebooks/tree/main/jupyter/minimal/ubi8-python-3.8`